include/ignition/utils/ExtraTestMacros.hh

```diff
@@ -17,3 +17,12 @@
 
 #include <gz/utils/ExtraTestMacros.hh>
 #include <ignition/utils/config.hh>
+
+#define IGN_UTILS_TEST_DISABLED_ON_WIN32(TestName) \
+  GZ_UTILS_TEST_ENABLED_ONLY_ON_LINUX(TestName)
+
+#define IGN_UTILS_TEST_DISABLED_ON_MAC(TestName) \
+  GZ_UTILS_TEST_ENABLED_ONLY_ON_LINUX(TestName)
+
+#define IGN_UTILS_TEST_ENABLED_ONLY_ON_LINUX(TestName) \
+  GZ_UTILS_TEST_ENABLED_ONLY_ON_LINUX(TestName)
```

Comment:
to simplify the merge process, I would move these declarations to the `gz/*` header files with a big comment saying it's temporary. this would allow them to be merged incrementally and then updated once they are all in. The move back to `ignition/` headers could be done in the next round of pull requests

Reply:
Ultimately, I think it was a mistake to do the header migration before the other changes. But now that we're in this situation, I'd like to reduce the number of rounds of PRs we need to do for all libraries. I'll coordinate with @methylDragon to make sure this is the last round of changes to all headers so we get this done with and can move on to other tasks.